src/sqlitestorage.h

```diff
@@ -672,9 +672,13 @@ class MKCAL_EXPORT SqliteStorage : public ExtendedStorage
 #define SELECT_COMPONENTS_BY_LAST_MODIFIED_AND_NOTEBOOK \
 "select * from Components where DateLastModified>=? and DateCreated<? and Notebook=? and DateDeleted=0"
 #define SELECT_COMPONENTS_BY_DELETED \
-"select * from Components where DateDeleted>=? and DateCreated<?"
+"select * from Components where DateDeleted<>0"
```

Comment:
2) ... Thus removing the DateCreated parameter will cause the query to grow over time;
If 'after' parameter is not valid, we will get empty list otherwise (as far as I understand), as we do not bind any value corresponding to 'after' parameter, and '>' operator on an unbound parameter always evaluates to UNKNOWN which converts to FALSE;
What do we need to use for 'DateCreated<?' in this case (when no valid 'after' is provided)?